Source/quests.cpp

```diff
@@ -647,7 +647,7 @@ void ResyncQuests()
 			ObjChangeMapResync(setpc_x, setpc_y, (setpc_w / 2) + setpc_x + 2, (setpc_h / 2) + setpc_y - 2);
 			for (int i = 0; i < nobjects; i++)
 				SyncObjectAnim(objectactive[i]);
-			int tren = TransVal;
+			int8_t tren = TransVal;

```

Comment:
```suggestion
			auto tren = TransVal;
```
TransVal is currently a char type, given this temp is purely to save and restore the initial state of TransVal it might as well use an inferred type.

Reply:
Really good point 🤔